pingdom/api_responses.go

```diff
@@ -82,6 +82,25 @@ type ProbeResponse struct {
 	Region     string `json:"region"`
 }
 
+// TeamResponse represents the json response for teams from the PIngdom API
+type TeamResponse struct {
+	ID    interface{} `json:"id"`
```

Comment:
The Pingdom documentation is inconsistent on whether IDs in team responses are integers or strings.  The documentation for (`Get Teams`, `Get Team`) show the ID as a string while (`Create Team`, `Update Team`) show the ID as an int.

However when I try this all 4 methods return a string for me.  Do you see that also?

Reply:
I think I saw int in return, but I will re-check.